Change "Product" to "product" (#3154)
theforeman/foreman-documentation: guides/common/modules/proc_importing-individual-iso-images-and-files.adoc

To use the CLI instead of the {ProjectWebUI}, see the xref:cli-importing-individual-iso-images-and-files[].

.Procedure

> . In the {ProjectWebUI}, navigate to *Content* > *Products*, and in the products window, click *Create Product*.

Reviewer: ```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Products*, and click *Create Product*.
```
We can probably remove this, because the user is clearly navigated.

. In the *Name* field, enter a name to identify the product.
This name populates the *Label* field.
. Optional: In the *GPG Key* field, enter a GPG Key for the product.
. Optional: From the *Sync Plan* list, select a synchronization plan for the product.
. Optional: In the *Description* field, enter a description of the product.
. Click *Save*.

> . In the products window, click the new product and then click *Create Repository*.

Reviewer: ```suggestion
. On the *Products* page, click the new product and then click *Create Repository*.
```
Web UI term?
